src/sentry/audit_log/events.py

```diff
@@ -12,7 +12,9 @@ def __init__(self):
     def render(self, audit_log_entry: AuditLogEntry):
         if audit_log_entry.target_user == audit_log_entry.actor:
             return "joined the organization"
-        return f"add member {audit_log_entry.target_user.get_display_name()}"
+
+        member = audit_log_entry.data.get("email") or audit_log_entry.target_user.get_display_name()
```

Comment:
Why don't you have target user in your case?

Reply:
I can look into adding it to the endpoint i just created, but there are a few endpoints currently, for example `OrganizationSCIMMemberDetails`, `OrganizationMember` that don't set the variable as well. We need to make this fix so that call so all audit logs with those entries actually render because they won't be currently 😬

Reply:
The user can be missing if the user was deleted, that will result in `target_user` being `None`.